README.md

```diff
@@ -1,15 +1,60 @@
-# Fresnel
+# fresnel
 
-Fresnel is a python library that ray traces publication quality images in real time.
+**fresnel** is a python library for path tracing publication quality images of soft matter simulations in real time.
+The fastest render performance is possible on NVIDIA GPUs using their [OptiX](https://developer.nvidia.com/optix)
+ray tracing engine. **fresnel** also supports multi-core CPUs using Intel's [Embree](https://embree.github.io/)
+ray tracing kernels. Path tracing enables high quality global illumination and advanced rendering effects controlled by
+intuitive parameters (like *roughness*, *specular*, and *metal*).
 
-Read the [tutorial and reference documentation on readthedocs](<https://fresnel.readthedocs.io/>).
+# fresnel Community
 
-## Samples
+Subscribe to the [fresnel-users](https://groups.google.com/d/forum/fresnel-users) mailing list to receive release
+announcements, post questions for advice on using the software, and discuss potential new features. File bug reports
+on [fresnel's issue tracker](https://github.com/glotzerlab/fresnel/issues). **fresnel** is an open source project.
+Please review [the contributor's guide](CONTRIBUTING.md) for more information before contributing.
+
+# Documentation
+
+Read the [tutorial and reference documentation on readthedocs](<https://fresnel.readthedocs.io/>). The tutorial
```

Comment:
I think the angle brackets should be removed from this link? It seems to work, but it's a nonstandard syntax in my understanding.